### PR TITLE
feat(cb2-8226): change in error message

### PR DESCRIPTION
**Pull request by emilyrkelly21**
## Email validation

- Applicant Email Address 
- Purchasers Email Address 
- Manufacturers Email Address 
**A valid email must contain:**
An “@”
A “.” after the “@”
[CB2-8226](https://dvsa.atlassian.net/browse/CB2-8226)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/forms/validators/custom-validators.ts [hunk 1]
```diff
@@ -180,7 +180,10 @@ export class CustomValidators {
   }
 
   static email(): ValidatorFn {
-    return this.customPattern(['^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$', 'is an invalid format. An email must include an @ symbol and a .']);
+    return this.customPattern([
+      '^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$',
+      'is an invalid format. An email must include an @ symbol and a . and end with two or more characters'
```
Comment (gjulien-bjss) [inline src/app/forms/validators/custom-validators.ts:185]:
```suggestion
      'is an invalid format. An email must include an @ symbol, include a full stop and end with two or more characters.'
```
(mostly there are two "and"s in the sentence)

Comment (gjulien-bjss) [reply src/app/forms/validators/custom-validators.ts:185]: the error message on develop does kinda look like a sentence that's not finished

Comment (andrewf-bjss) [inline src/app/forms/validators/custom-validators.ts]:
```suggestion
    return this.customPattern([
      '^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$',
      'is an invalid format. An email must include an @ symbol, a full-stop, and end with two or more characters'
```

File: src/app/forms/validators/custom-validators.ts [hunk 2]
```diff
@@ -180,7 +180,10 @@ export class CustomValidators {
   }
 
   static email(): ValidatorFn {
-    return this.customPattern(['^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$', 'is an invalid format. An email must include an @ symbol and a .']);
+    return this.customPattern([
+      '^[-\\w.\\+]+@[-\\w]+\\.[A-Za-z]{2,}$',
+      'is an invalid format. An email must include an @ symbol, include a full stop and end with two or more characters.'
```
Comment (andrewf-bjss) [inline src/app/forms/validators/custom-validators.ts:185]: has `include` in it twice now... how about

```suggestion
      'is an invalid format. An email must include an @ symbol, a full stop and end with two or more characters'
```

File: src/app/forms/validators/custom-validators.ts (hunk 1 above)
Comment (andrewf-bjss) [reply src/app/forms/validators/custom-validators.ts]: outdated - resolving.

**Review by andrewf-bjss — APPROVED** [review]

**Review by kcarr-bjss — APPROVED** [review]

**Pull request merged**